servicetalk-examples/http/jaxrs/src/main/java/io/servicetalk/examples/http/jaxrs/HelloWorldExceptionMapper.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright © 2023 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.examples.http.jaxrs;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+public class HelloWorldExceptionMapper implements ExceptionMapper<Throwable> {
+    @Override
+    public Response toResponse(final Throwable exception) {
+        exception.printStackTrace();
+        return Response.serverError().build();
```

Comment:
They way current example is placed is hard for users to run it and see the difference. Consider adding 2 endpoints in `HelloWorldJaxRsResource` that throw different exception types. Add proper javadoc to tell users how to run a `curl` command for each endpoint and what is expected result. Then adjust `HelloWorldExceptionMapper` to target only one of these 2 exceptions and map it to a different status code compare to the default Jersey behavior. That way users can execute 2 curl commands and observe the difference.